File: aoj/volume100/java/A10025.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double a = scan.nextDouble();
        double b = scan.nextDouble();
        double rad = Math.toRadians(scan.nextDouble());
        double c = Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2) - 2*a*b*Math.cos(rad));

        double area = a*b*Math.sin(rad)/2.0;
        double perimeter = a + b + c;
        double h = 2*area / a;
        System.out.println(area);
        System.out.println(perimeter);
        System.out.println(h);
    }
}

File: aoj/volume100/php/A10025.php
<?php
fscanf(STDIN, "%f %f %f", $a, $b, $deg_c);

$rad_c = $deg_c * pi() / 180.0;
$c = sqrt(pow($a, 2) + pow($b, 2) - 2*$a*$b*cos($rad_c));

$area      = $a * $b * sin($rad_c) / 2.0;
$perimeter = $a + $b + $c;
$h         = 2 * $area / $a;

printf("%.6f\n%.6f\n%.6f\n", $area, $perimeter, $h);
